src/Functions/CMakeLists.txt

```diff
@@ -6,6 +6,14 @@ add_headers_and_sources(clickhouse_functions .)
 list(REMOVE_ITEM clickhouse_functions_sources IFunctionImpl.cpp FunctionFactory.cpp FunctionHelpers.cpp)
 list(REMOVE_ITEM clickhouse_functions_headers IFunctionImpl.h FunctionFactory.h FunctionHelpers.h)
 
+add_custom_target(generate-error-codes
```

Comment:
Arcadia build system (ya.make) theoretially supports generated source files but nobody outside of Yandex (and also no one from ClickHouse developers) can master it.

It's better to either:
- disable this function from Arcadia;
- don't use generated files;
- commit generated files to the repository.